WooCommerce/UITestsFoundation/Screens/MyStore/MyStoreScreen.swift

```diff
@@ -27,4 +27,46 @@ public final class MyStoreScreen: ScreenObject {
         topBannerCloseButton.tap()
         return self
     }
+
+    func tapTimeframeTab(timeframe: String) -> MyStoreScreen {
+        app.cells.staticTexts[timeframe].tap()
```

Comment:
WDYT of using the parent cell identifier instead? Example for `This Month`:

<img width="571" alt="Screenshot 2022-12-05 at 15 15 28" src="https://user-images.githubusercontent.com/73365754/205649655-bec1be30-d76e-4710-bfdf-0b816b88a429.png">

This will make the test less dependent on the localization during tab selection (you will still have a human-readable interval mentioned in the assertion of the chart hint text).

```suggestion
    func tapTimeframeTab(timeframeID: String) -> MyStoreScreen {
        app.cells[timeframeID].tap()
```

The respective values for IDs to use in calling functions would be:
- `period-data-today-tab`
- `period-data-thisWeek-tab`
- `period-data-thisMonth-tab`
- `period-data-thisYear-tab`

Reply:
ah yup i agree with your point, this is updated in https://github.com/woocommerce/woocommerce-ios/pull/8310/commits/27a1d7c2ea79251ad3e836ffa68ba2db740c24b1

Reply:
Thanks for addressing this @jostnes!